### PR TITLE
Fix #134 - Upgrade now modal

### PR DESCRIPTION
**Pull request by nico3333fr**
Spotted by design: some inconsistency were present.

- fixed some alignments
- centered image
- made link same color on hover
- updated icons
- fix position of close button
- remove CSS, used Design system classes
- added text to improve accessibility of close button

image
(approved by design :) )

Fixes: https://github.com/ProtonMail/proton-mail-settings/issues/134

### CODE REVIEW

File: containers/payments/subscription/FeaturesList.js [hunk 1]
```diff
@@ -7,15 +7,13 @@ const FeaturesList = ({ features = [] }) => {
...
             {features.map((text, index) => {
                 const key = `${index}`;
                 return (
-                    <li key={key} className="aligncenter pl0-5 pr0-5">
-                        <div>
-                            <Icon name="add" size={25} className="fill-pm-blue" />
-                        </div>
-                        {text}
+                    <li key={key} className="aligncenter flex-item-fluid pl1 pr1">
+                        <Icon name="check-circle" size={25} className="fill-pm-blue" />
```
Comment (EpokK) [inline containers/payments/subscription/FeaturesList.js]: Can we use `primary` color here?
```suggestion
                        <Icon name="check-circle" size={25} className="fill-pm-primary" />
```

Comment (nico3333fr) [reply containers/payments/subscription/FeaturesList.js]: Good catch, my mistake ^^

**Comment by nico3333fr** [comment]
@EpokK it will look like this with less items:

image

image

File: containers/payments/subscription/UpgradeModal.js [hunk 2]
```diff
@@ -22,50 +21,54 @@ const UpgradeModal = ({ plans, onUpgrade, onComparePlans, ...rest }) => {
...
+                <div className="pl2 pr2 pb2">
+                    <div className="flex-autogrid onmobile-flex-column">
+                        <div className="flex-autogrid-item flex flex-column flex-spacebetween">
+                            <h3 className="bold">{c('Title').t`Upgrade now!`}</h3>
+                            <div className="mb2">
+                                <div>{c('Info').t`Unlock additional features with`}</div>
+                                <div>
+                                    {c('Info').t`ProtonMail Plus for as low as`}{' '}
+                                    <PlanPrice
+                                        className="bold color-pm-blue"
```
Comment (EpokK) [inline containers/payments/subscription/UpgradeModal.js]: `primary`?

Comment (nico3333fr) [reply containers/payments/subscription/UpgradeModal.js]: Argh, shame on me.

4vI10

Comment (nico3333fr) [reply containers/payments/subscription/UpgradeModal.js]: Done :) https://github.com/ProtonMail/react-components/pull/308/commits/c601af1359e7111d7e46e7d956894dc2907c00a4

**Pull request merged**